### PR TITLE
Create JonasWagner.ttl

### PR DESCRIPTION
**Pull request by zumpious**

### CODE REVIEW

File: data/people/JonasWagner.ttl [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+dice:ExamplePerson a schema:Person ;
+  schema:name "Jonas Wagner" ;
+  schema:namePrefix "" ;
+  schema:role dice:ResearchStaff ;
+  schema:email <mailto:jonas.wagner@uni-paderborn.de> ;
+  schema:chat "@jwagner6:chat.dice-research.org" ;
+  schema:office "" ;
+  schema:photo "wagner.jpg" ;
+  schema:publicationTag "" ;
+  schema:content """
```
Comment (bigerl) [inline data/people/JonasWagner.ttl:13]:
```suggestion
  schema:publicationTag "" ;
  schema:content """
```
```suggestion
  schema:photo "wagner.jpg" . 
```

**Review by bigerl — CHANGES_REQUESTED** [review]
Hi Jonas,
your RDF was malformed. I've suggested a fix. Please ask your supervisor afterwards to squash-merge the changes.

**Comment by zumpious** [comment]
Hi @Perevalov , can squash-merge my PR?

**Comment by MichaelRoeder** [comment]
Hi Jonas,

You may want to double check your picture. I would suggest to cut the right and lower part a little bit, so that your face is more or less in the center of the picture (but still has some space around it). The page will later on tell your browser to form a circle on top of the picture (see https://dice-research.org/team/), which may look a little bit odd with your current picture (although I didn't try it :sweat_smile: ).

When you are done (or you are happy with the picture as it is), please ping me and I will merge your changes :wink:

**Pull request merged**